examples/plugin/plugin/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 # Copyright The OpenTelemetry Authors
 # SPDX-License-Identifier: Apache-2.0
 
+if(OPENTELEMETRY_SKIP_DYNAMIC_LOADING_TESTS)
```

Comment:
Probably better to check this in the parent CMakeList.txt, around `add_subdirectory(plugin)`?

Reply:
I updated the PR to do it this way.